libs/article-parser/src/lib/parser/ebi-parser.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ParsedArticle,
+  Parser,
+  ParsedArticleParagraph,
+  EbiParserOptions,
+} from '@foodmedicine/interfaces';
+import * as cheerio from 'cheerio';
+
+/**
+ * A parser for https://www.ebi.ac.uk/europepmc/webservices/rest/
+ */
+export const EbiParser: Parser<ParsedArticle> = {
+  parserF: async (xml: string, opts?: EbiParserOptions) => {
+    if (
+      !opts ||
+      !opts.parsedArticleHead ||
+      !opts.impacted ||
+      !opts.recommendation
+    ) {
+      throw 'Please add in the parsed head, impacted, and recommendation through the options';
+    }
+    const $ = cheerio.load(xml);
```

Comment:
Unless this is a convention for cheerio, please choose another variable name.  This is commonly associated with jQuery and could lead to some confusion

Reply:
Ok sure, I think it is convention as seen on the [Cheerio site](https://cheerio.js.org/)

Reply:
ok, ignore then